pkg/cloudprovider/providers/gce/gce_backendservice.go

```diff
@@ -69,6 +85,17 @@ func (gce *GCECloud) CreateGlobalBackendService(bg *compute.BackendService) erro
 	return gce.waitForGlobalOp(op, mc)
 }
 
+// CreateAlphaGlobalBackendService creates the given alpha BackendService.
+func (gce *GCECloud) CreateAlphaGlobalBackendService(bg *computealpha.BackendService) error {
+	mc := newBackendServiceMetricContextWithVersion("alpha_create", "", computeAlphaVersion)
```

Comment:
having both the name alpha_xxx and computeAlphaVersion in the metric is overkill. We should just have one or the other.